packages/gatsby/src/utils/api-node-docs.ts

```diff
@@ -431,6 +431,7 @@ export const onCreateDevServer = true
 
 /**
  * Called during `gatsby develop` bootstrap to get and validate a plugins options schema
+ * @gatsbyVersion 2.24.69
```

Comment:
Oh wow, is the PR only that size? How is it working under the hood?

Reply:
Yep! @DSchau implemented this way back in 2019 in #16105, I just didn't know! Soooo goooood 😍

Reply:
Oh wow, this is an incredible feature! Thanks for pointing the PR, will study on that 🚀

Reply:
@mxstbr what do you think of arbitrarily bumping the version of Gatsby here? (e.g. to 2.25.0 or something)

Seems better user behavior (cc @vladar useful to think about as a concrete example of release cadence being useful!)

Reply:
Oh -- I mean bump the version locally and _then_ ship this 🚀

Reply:
Let's bump that version after we land #27381 and set the minimum version to that!